src/GraphScene.tsx

```diff
@@ -43,6 +45,11 @@ export interface GraphSceneProps {
    */
   actives?: string[];
 
+  /**
+   * List of node ids that are expanded.
+   */
+  expandedParents?: string[];
```

Comment:
Lets invert this logic to:

```
collapsedNodeIds?: string[]
```

so that everything is expanded by default unless you specify otherwise and that should remove the need for the other `expanded` flag too.